clients/js/README.md

```diff
@@ -31,6 +36,7 @@ Commands:
   worm near                                       NEAR utilities
   worm submit <vaa>                               Execute a VAA
   worm update                                     Update this tool by rebuilding it
+  worm verify-vaa --vaa <vaa> --network <network> Verifies a VAA by querying the core contract on Ethereum
```

Comment:
How hard is it to write a generic version for all supported chains?

Reply:
This command is meant to verify the VAA rather than the chain. It just does the verification on Ethereum because that is (probably) the easiest. If the VAA is valid on Ethereum, it should be valid throughout the entire wormhole network. I don't think there is any need or advantage to doing this on multiple chains.